test/quantization/pt2e/test_quantize_pt2e.py

```diff
@@ -1814,6 +1814,22 @@ def forward(self, x, y):
             non_ref_node_occurrence={}
         )
 
+    def test_representation_maxpool2d(self):
+        quantizer = XNNPACKQuantizer()
+        operator_config = get_symmetric_quantization_config(is_per_channel=True)
+        quantizer.set_global(operator_config)
+        m_eager = TestHelperModules.ConvMaxPool2d().eval()
+
+        example_inputs = (torch.randn(1, 2, 2, 2),)
```

Comment:
Is the replacement failing here because of "burnt" in scalars issue?

Reply:
the replacement succeeds in the test (checked locally), do you mean why the replacement doesn't work before we have `replace_literals_with_placeholders` workaround?